Gemfile

```diff
@@ -44,7 +44,7 @@ end
 if (puppetversion = ENV['PUPPET_GEM_VERSION'])
```

Comment:
we shouldn't need a version specified here at all. It will use the installed version or install the newest version if none is specified, which is what we want.

```ruby
if puppetversion = ENV['PUPPET_GEM_VERSION']
  gem 'puppet', puppetversion, :require => false
else
  gem 'puppet', :require => false
end
```